### PR TITLE
mig527: Velero debug tool

### PR DESCRIPTION
**Pull request by apinnick**
https://issues.redhat.com/browse/MIG-527

### CODE REVIEW

File: modules/migration-debugging-velero-resources.adoc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+// * migration/migrating_3_4/troubleshooting-3-4.adoc
+// * migration/migrating_4_1_4/troubleshooting-4-1-4.adoc
+// * migration/migrating_4_2_4/troubleshooting-4-2-4.adoc
+
+[id="migration-debugging-velero-resources_{context}"]
+= Using the Velero CLI to debug Backup and Restore CRs
+
+You can debug the `Backup` and `Restore` custom resources (CRs) and partial migration failures with the Velero command line interface (CLI). The Velero CLI runs in the `velero` pod.
+
+== Velero command syntax
```
Comment (ahardin-rh) [inline modules/migration-debugging-velero-resources.adoc:11]: Each of the level 2 headings must have a unique section ID associated with it (Velero command syntax, Help command, etc.)

**Comment by ahardin-rh** [comment]
Just one comment from me. Otherwise, looks good!

File: modules/migration-debugging-velero-resources.adoc (hunk 1 above)
Comment (apinnick) [reply modules/migration-debugging-velero-resources.adoc:11]: Oops -- yes, I forgot to do that. Will fix.

**Pull request merged**